ubuntu/intel/uptodate.yaml

```diff
@@ -2,8 +2,9 @@ dockerbuild:
   build_args:
     intel_version:
       key: intel
-      versions:
-        - "2021.2.0"
+      name: intel-oneapi-compilers
```

Comment:
How is this being passed into the Dockerfile?

Reply:
oh nevermind, you are updating an existing one right?

Reply:
Yup, the `intel` key is used in the existing Dockerfile to drive the Spack install. This change means the versions will come from the Spack package, rather than the list.

Reply:
@davidbeckingsale I see this building 2021.4 which is the latest any chance of getting a 2022 build or is this a limitation of the spack packages for oneapi?

Reply:
It should be finding the versions in the Spack package, which has up to 2022.0.1 - let me figure out what's going on.

Reply:
I don't see 2022 in versions https://spack.github.io/packages/package.html?name=intel-oneapi-compilers

Reply:
I'm asking in the Spack slack...

Reply:
I think someone turned off the automated workflow https://github.com/spack/packages without telling me. :(

Reply:
ug it was GitHub!! Poozers. 
![image](https://user-images.githubusercontent.com/814322/150011407-c2a75f5c-5a4f-42bf-96c4-8ae2229e048f.png)

Reply:
I see the following on the "Action" tab:

> This scheduled workflow is disabled because there hasn't been activity in this repository for at least 60 days. Enable this workflow to resume scheduled runs.

Reply:
I'm re-running it now. It's weird normally I get an email notification but because this isn't my org I didn't.

Reply:
We need github actions to update our actions